cime_config/allactive/config_compsets.xml

```diff
@@ -86,6 +86,88 @@
   <lname>20TRSOI_EAM%CMIP6_ELM%CNPRDCTCBCTOP_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
 </compset>
 
+<!-- Compsets for historical single forcing experiments, two aliases each with WCYCL20TR or hist as prefix -->
+
+<compset>
+  <alias>WCYCL20TR-GHG</alias>
+  <lname>20TRSOI_EAM%CMIP6-GHG_ELM%CNPRDCTCBCTOP_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
+</compset>
+
+<compset>
+  <alias>hist-GHG</alias>
```

Comment:
Why are you introducing aliases that start with "hist" ?

Reply:
I'd like those removed.   Our convention is that "20TR" means a "historical" run.  We should not introduce a second competing convention.  Especially since it requires entering the same compset twice.  Those could get out-of-sync.

Reply:
@rljacob The simplied 'hist-xxxx' format is more consistent with the DAMIP configuration name. Yes, we have been using WCYCL20TR format for historical. We can have discussion if it is useful to also have such alternative aliases. BTW, the alias names are not final and expected to be revised (note some of the config name listed by @golaz are different).